.github/workflows/python-release.yml

```diff
@@ -84,7 +84,16 @@ jobs:
         if: startsWith(matrix.os, 'ubuntu')
         run: ls -lah dist/* && cp dist/* wheelhouse/
 
-      - uses: actions/upload-artifact@v3
+      - uses: actions/upload-artifact@v4
         with:
-          name: "release-${{ github.event.inputs.version }}"
+          name: "release-${{ matrix.os }}"
           path: ./wheelhouse/*
+  merge:
```

Comment:
Ohh, this is nice! https://github.com/actions/upload-artifact/blob/main/merge/README.md